Yank/experiment.py

```diff
@@ -1657,16 +1659,21 @@ def _is_pipeline_solvent_with_receptor(field, solvent_id, error):
             schema:
                 type: string
                 validator: file_exists
-            dependencies: phase2_path
```

Comment:
Why did you remove the dependency on `phase2_path` here?

Reply:
For the records, we talked about this, and we decided to postpone trying to rework the schema to use `required: yes` and `excludes: [receptor, ligand, solute]` in a future PR.

I'll re-add the `dependencies` solution here before merging!